curated_transformers/layers/attention.py

```diff
@@ -254,25 +383,46 @@ def __init__(self, *, n_query_heads: int, n_key_value_heads: int):
             Number of query heads.
         :param n_key_value_heads:
             Number of key/value heads.
+        :param qkv_split:
+            How query, key, and value should be split when using
+            :py:class:`~curated_transformers.layers.QkvMode.MERGED_SPLIT_AFTER`.
+            Not used for other query, key, value modes.
 
         :meta private:
         """
         self._n_query_heads = n_query_heads
         self._n_key_value_heads = n_key_value_heads
+        self._qkv_split = QkvSplitKVSizedChunks() if qkv_split is Default else qkv_split
```

Comment:
No, our default for API backwards compatibility is `QkvSplitKVSizedChunks` because we did Falcon first. So we heed a better name for `QkvSplitDefault` 🤔 .

Reply:
Renamed as discussed in Slack.